ONSdigital/go-ns
# Allow florence token to be sent over as http cookie in auth clients

## What

There is a requirement for some services (e.g the download service) to accept a florence token as a cookie with name `access_token`. The client auth package has been extended to allow for this

## How to review

- Verify Logic is correct
- Run tests

## Who can review

Anyone
